components/Elevation/src/UIView+MaterialElevationResponding.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+/**
+ Allows elevation changes to propagate down the view hierarchy and allows objects
+ conforming to @c MDCElevation to react to those changes accordingly.
+ */
+@interface UIView (MaterialElevationResponding)
+
+/**
+ Returns the sum of all @c superviews in the view hierarchy that conform to @c MDCElevation @c
```

Comment:
Tip: Suggest changing to something that clarifies a bit more about what kind of math is done.  Something like,  "Returns the sum of the receiver's `mdc_currentElevation` and its' superview's `mdc_baseElevation`.  If the superview responds to `mdc_overrideBaseElevation`, returns the sum of that and the receiver's `mdc_currentElevation`."

Reply:
It doesn't account for `mdc_currentElevation` its only the elevation of `superview.mdc_currentElevation` recursively.

Reply:
Updated.